pluginsdk/Microsoft.Windows.DevHome.SDK/Microsoft.Windows.DevHome.SDK.idl

```diff
@@ -111,7 +113,7 @@ namespace Microsoft.Windows.DevHome.SDK
 
         event Windows.Foundation.TypedEventHandler<IDeveloperIdProvider, Object> Changed;
 
-        AuthenticationState developerIDState; 
+        AuthenticationState DeveloperIdState; 
```

Comment:
Did we settle on whether DeveloperIdState is a member od DeveloperIdProvider or DeveloperId?

Reply:
This is currently based on the initial API design reviews. We can also confirm today at the meeting. Thanks!